data/data-pipeline/data_pipeline/ipython/aggregate_burden_exploration.ipynb

```diff
@@ -0,0 +1,1432 @@
+    {
+     "data": {
+      "text/plain": [
+       "pct_cal_score_0.9  DRAFT CES 4.0\\nPercentile Range\n",
+       "False              10-15%                             0.052534\n",
+       "                   20-25%                             0.052534\n",
+       "                   40-45%                             0.052534\n",
+       "                   1-5% (lowest scores)               0.052402\n",
+       "                   15-20%                             0.052402\n",
+       "                   25-30%                             0.052402\n",
+       "                   30-35%                             0.052402\n",
+       "                   35-40%                             0.052402\n",
+       "                   5-10%                              0.052402\n",
+       "                   45-50%                             0.052269\n",
+       "                   50-55%                             0.052269\n",
+       "                   60-65%                             0.052137\n",
+       "                   55-60%                             0.052005\n",
+       "                   65-70%                             0.051343\n",
+       "                   70-75%                             0.050417\n",
+       "                   75-80%                             0.048697\n",
+       "                   80-85%                             0.047506\n",
+       "                   85-90%                             0.041419\n",
+       "                   90-95%                             0.037846\n",
+       "                   95-100% (highest scores)           0.028847\n",
+       "                   NaN                                0.013233\n",
+       "True               95-100% (highest scores)           0.374477\n",
+       "                   90-95%                             0.232218\n",
+       "                   85-90%                             0.173640\n",
+       "                   80-85%                             0.079498\n",
+       "                   75-80%                             0.058577\n",
+       "                   70-75%                             0.033473\n",
+       "                   65-70%                             0.016736\n",
+       "                   NaN                                0.010460\n",
+       "                   55-60%                             0.006276\n",
+       "                   60-65%                             0.006276\n",
+       "                   50-55%                             0.004184\n",
+       "                   30-35%                             0.002092\n",
+       "                   45-50%                             0.002092\n",
+       "Name: DRAFT CES 4.0\\nPercentile Range, dtype: float64"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ces_merged.groupby(\"pct_cal_score_0.9\")[\"DRAFT CES 4.0\\nPercentile Range\"].value_counts(\n",
```

Comment:
Nit: could you maybe add a comment explaining verbally how to interpret these results? E.g., "Each row of the following outputs answers the question, out of communities that are or are not in the top 10% of CalEnviroScreen, ...".

Reply:
Sure